packages/cli/bin/gridsome.js

```diff
@@ -59,8 +59,9 @@ program.arguments('<command>').action(async command => {
     const useYarn = await hasYarn()
 
     console.log()
-    console.log(`  Please run ${chalk.cyan(useYarn ? 'yarn' : 'npm install')} to install dependencies first.`)
+    console.log(chalk.red(`Unknown command ${chalk.bold(command)}`))
```

Comment:
Why not taking it outside the `if/else` block?

Reply:
```js
program.arguments('<command>').action(async command => {
  const { isGridsomeProject, hasYarn } = require('../lib/utils')
  const availableCommands = program.commands.map(cmd => cmd._name)
  const suggestion = availableCommands.find(cmd => {
    const steps = leven(cmd, command)
    return steps < 3
  })

  console.log(chalk.red(`Unknown command ${chalk.bold(command)}`))

  if (isGridsomeProject(pkgPath) && !suggestion) {
    const useYarn = await hasYarn()

    console.log()
    console.log()
    program.outputHelp()
  } else if (suggestion) {
      console.log()
      console.log(`Did you mean ${suggestion}?`)
  }
})
```

Reply:
Yes, that seems like a better solution @jamesgeorge007 :)

Reply:
@jamesgeorge007 Cheers for that. Agree. Btw, is the change to `suggestion` intentional?

```js
const suggestion = didYouMean(command, availableCommands)
```

```js
const suggestion = availableCommands.find(cmd => {
    const steps = leven(cmd, command)
    return steps < 3
  })
```

Reply:
#1016

Reply:
#1001